Services/DataX.Flow/DataX.Flow.InteractiveQuery/InteractiveQueryManager.cs

```diff
@@ -37,7 +38,47 @@ public InteractiveQueryManager(ILogger logger, IConfiguration configuration)
             _engineEnvironment = new EngineEnvironment(_configuration);
         }
 
-        private string SetupSteps { get; set; } = string.Empty;
+        private string SetupSteps { get; set; } = string.Empty;
+
+        /// <summary>
+        /// This method converts wasbs path to dbfs file path
+        /// </summary>
+        /// <param name="filePath">wasbs file path</param>
+        /// <param name="fileName">file name</param>
+        /// <returns>Returns dbfs file path</returns>
+        public static string ConvertToDbfsFilePath(string filePath, string fileName = "")

```

Comment:
Looks like the better place for these new methods is DataX.Flow.Common.Helper. Thoughts?

Reply:
moved